### DISCUSSION TITLE
升级之后list文件列表速度超慢

### DISCUSSION THREAD

**Discussion by xxxzsgxxx** [houtianze/bypy#670] [General]
之前版本1.8.2速度还基本正常，升级到1.8.4之后，ls文件和目录列表时候，速度非常非常慢，但也能出来。

**Comment by houtianze** [comment]
帮忙贴下`bypy -dv ls <remote dir>`的输出？这次改动只是秒传相关的地方，所以不确定为啥会影响列表。

**Reply by xxxzsgxxx** [reply]
长时间卡在了这个地方：
<D> Error occurred while checking update, skipping.

**Comment by houtianze** [comment]
你试一下用浏览器能打开这个链接吗？ https://raw.githubusercontent.com/houtianze/bypy/master/update/update.json

**Reply by xxxzsgxxx** [reply]
默认，不可以打开。需要科学之后才可以打开。

**Comment by houtianze** [comment]
科学改变世界。。那帮我试一下换成`http`（没s）协议能打开不？能打开的画我后面把链接改一下。

**Comment by xxxzsgxxx** [comment]
> 科学改变世界。。那帮我试一下换成`http`（没s）协议能打开不？能打开的画我后面把链接改一下。

可以打开，现在速度也快了很多。不卡顿了，谢谢。
